www/js/tablet.js

```diff
@@ -376,22 +389,7 @@ function tabletShowMessage(msg, collecting) {
     initialGuess.br.x = parseFloat(msg.substring(13, msg.length))
     return;
   }
-  if (msg.startsWith('$/Maslow_tlZ=')) {
```

Comment:
I missed this at first and just found it while working on something else, but it looks like these got deleted by accident

Reply:
yikes, yes, I certainly didn't want that to be removed. Good catch!